### PR TITLE
Missing link

### PR DESCRIPTION
**Pull request by migig**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@migig__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: articles/app-service-mobile/app-service-mobile-offline-data-sync.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ The following tutorials show how to add offline sync to your mobile clients usin
 * [iOS: Enable offline sync]
 * [Xamarin iOS: Enable offline sync]
 * [Xamarin Android: Enable offline sync]
+* [Xamarin.Forms: Enable offline sync](http://azure.microsoft.com/en-us/documentation/articles/app-service-mobile-xamarin-forms-get-started-offline-data/)
```
Comment (wesmc7777) [inline articles/app-service-mobile/app-service-mobile-offline-data-sync.md]: Can you change this to ...

[Xamarin.Forms: Enable offline sync](app-service-mobile-xamarin-forms-get-started-offline-data.md)

Comment (migig) [inline articles/app-service-mobile/app-service-mobile-offline-data-sync.md]: Done

**Comment by wesmc7777** [comment]
#sign-off

**Pull request merged**